tests/test_angleunit.py

```diff
@@ -52,8 +52,11 @@ def test_init():
 def test_builtin_units():
     """Check the built-in AngleUnits
     """
-    pass
-
+    angle_rad = coord.AngleUnit(pi)
+    assert angle_rad.value == 180.*coord.degrees.value
```

Comment:
I think this might be clearer as e.g.
```
assert coord.degrees.value == pi/180.
```
rather than doing this via the `angle_rad.value`, which is really just another name for `pi` here.